### PR TITLE
Fix learning rate decay in ElasticNet attack

### LINKED ISSUE #1829: ElasticNet attack uses quadratic learning rate decay instead of square root
https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/8aa8c01b987c60aaca0cff3e4b2af5098218dce2/art/attacks/evasion/elastic_net.py#L193

According to [the paper](https://arxiv.org/pdf/1709.04114.pdf), I believe this should be:

```python
decayed_learning_rate = learn_rate * (1 - global_step / decay_steps) ** 0.5 + end_learning_rate 
```

### PR DESCRIPTION
**Pull request by beat-buesser**
Signed-off-by: Beat Buesser <beat.buesser@ie.ibm.com>

# Description

This pull request fixes the learning rate decay in ElasticNet Attack.

Fixes #1829 

## Type of change

Please check all relevant options.

- [ ] Improvement (non-breaking)
- [x] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: tests/attacks/test_elastic_net.py [hunk 1]
```diff
@@ -138,6 +138,7 @@ def test_4_tensorflow_mnist(self):
                 0.0,
             ]
         )
+        print(x_test_adv[0, 14, :, 0])
```
Comment (Zaid-Hameed) [inline tests/attacks/test_elastic_net.py:141]: @beat-buesser maybe you can remove this and other print statements in the code.

**Review by Zaid-Hameed — APPROVED** [review]
It is ready to merge once print statements in test_elastic_net.py are removed.

File: tests/attacks/test_elastic_net.py (hunk 1 above)
Comment (beat-buesser) [reply tests/attacks/test_elastic_net.py:141]: I agree.
